starboard/linux/shared/platform_configuration/configuration.gni

```diff
@@ -41,3 +41,5 @@ platform_tests_path =
 enable_in_app_dial = true
 
 v8_enable_webassembly = true
+
+sb_enable_cast_codec_tests = true
```

Comment:
I'd recommend against this as it's pretty brittle when the arg isn't declared in base_configuration.gni

Reply:
Okay - but what do you recommend doing instead ? 

Note the timing of getting _a version_ of this into the code is important. If there's a better way to do this without adding a lot of extra code, maybe we can follow up from here.